app/controllers/podcasts_controller.rb

```diff
@@ -130,4 +132,17 @@ def podcast_params
       ]
     )
   end
+
+  def compose_message(podcast)
+    title = "<#{podcast_url(podcast.id)}|#{sub_escapes(podcast.title)}>"
+    intro = "A new podcast has been created: #{title}"
+    id = "ID: #{podcast.id}"
+    planner = "<#{podcast_planner_url(podcast.id)}|Schedule episodes>"
+    imports = "<#{podcast_imports_url(podcast.id)}|Start a new import>"
+    [intro, id, planner, imports].join("\n")
```

Comment:
wasn't sure what all was going to be helpful for distro, so i included the planner and the importer links if i were guessing where someone might need to go next.

Reply:
I think this is a good start.  We'll find out!